server/server/src/wsserver.ts

```diff
@@ -22,18 +22,25 @@ import { assignWorkspace, closeWorkspace, WorkspaceInfo } from './workspaces'
 /**
  * @public
  */
-export async function startServer (host: string, port: number, serverToken: string): Promise<Server> {
+export interface ServerOptions {
+  logTransactions?: boolean
+  logRequests?: boolean
+}
+/**
+ * @public
+ */
+export async function startServer (host: string, port: number, serverToken: string, options?: ServerOptions): Promise<Server> {
   const instance = await start(host, port, {
-    connect: connectClient(serverToken),
+    connect: connectClient(serverToken, options),
     close: async (clientId) => {
       await closeWorkspace(clientId)
     }
   })
   return instance
 }
 function connectClient (
-  serverToken: string
-): (clientId: string, token: string, sendTx: (tx: Tx) => void, close: () => void) => Promise<Storage> {
+  serverToken: string, options?: ServerOptions
```

Comment:
I'd suggest to set default options:
```ts
options: ServerOptions = {logRequests: false, logTransactions: false}
```

Reply:
For me it is same as not defined. Will fix if more options will occur.

Reply:
Fixed